CONTRIBUTING.md

```diff
@@ -153,6 +153,22 @@ At this point the pre-commit checks will run against any files that
 you attempt to commit.  If you want to run the checks against the
 entire repo, just execute `pre-commit run --all-files`.
 
+## Quality assurance/code reviews ##
+
+In order to maintain standardization with practices, ensure security standards
+are being met, and to incorporate third party code as seamlessly as possible,
+submitted code will go through our quality assurance team.
+
+Code contributors are able to coordinate with the quality assurance (QA) team
+at any point in their projects. We would recommend initiating the discussions
+as early as possible, to decrease the likelihood of blockers to merging or
+using their code occurring later in the process. However, the QA team is not
+responsible for the success of the project or ensuring all team members follow
+the development standards which have been established. Any discussions or
+initial input is a courtesy evaluation, where contributing teams remain
+responsible for the quality of their code, internal coordination,
+and alignment with the standards set forth in this guide.
```

Comment:
Grammatical things and minor language changes to make it read more smoothly.
```suggestion
## Quality assurance and code reviews ##

In order to maintain standardization of practices, ensure security standards
are being met, and incorporate third party code as seamlessly as possible,
submitted code will go through our quality assurance (QA) team.

Code contributors are able to coordinate with the quality assurance team
at any point during their projects. We recommend initiating the discussions
as early as possible, to decrease the likelihood of issues around merging or
using the contributed code occurring late in the process. However, the QA team is not
responsible for the success of the project or for ensuring that all team members follow
the development standards which have been established. Any discussions or
initial input are a courtesy evaluation, and contributing teams remain
responsible for the quality of their code, internal coordination,
and alignment with the standards set forth in this guide.
```

I still feel like this section addresses large scale contributors ("I'm want to create a new `cisagov` repo") but does not address/apply to small scale contributors ("I want to fix a typo in the documentation in an existing `cisagov` repo" or "I want to add an additional test to an existing `cisagov` repo").

Reply:
Resolved in commit f34b45e.